src/web/vaev-layout/table.cpp

```diff
@@ -845,6 +861,12 @@ struct TableFormatingContext : public FormatingContext {
         rowHelper = buildAxisHelper(rows, rowGroups, grid.size.y);
         colHelper = buildAxisHelper(cols, colGroups, grid.size.x);
 
+        startPositionOfRow = Buf<Px>::init(grid.size.y, 0_px);
```

Comment:
You are forcing an allocation here. Better do the following to give a chance to Vec<> to reuse the buffer it might have already allocated.

```c++
startPositionOfRow.clear();
startPositionOfRow.resize(grid.size.y, 0_px);
```